templates/cli/lib/parser.js.twig

```diff
@@ -110,13 +114,40 @@ const drawJSON = (data) => {
 }
 
 const parseError = (err) => {
-    if(cliConfig.verbose) {
-        console.error(err);
-    }
+    if (cliConfig.report) {
+        (async () => {
+            let appwriteVersion = 'unknown';
+            const isCloud = globalConfig.getEndpoint().includes('cloud.appwrite.io') ? 'Yes' : 'No';
+
+            try {
+                const res = await fetch(`${globalConfig.getEndpoint()}/health/version`);
+                const json = await res.json();
+                appwriteVersion = json.version;
+            } catch {
+            }
+
+            const version = '0.16.0';
```

Comment:
Is this SDK version? Can we see what we do in `version` command and copy that, so it auto-updates?

Reply:
That's mistake, now it's auto-generated